packages/components-js/projects/components-manager-cli/library-entry/with-prefix.spec.ts

```diff
@@ -65,45 +65,93 @@ describe('load()', () => {
+    it('should for window.PORSCHE_DESIGN_SYSTEM_CDN="auto" set document.porscheDesignSystem.cdn.url="https://cdn.ui.porsche.com"', () => {
+      window.PORSCHE_DESIGN_SYSTEM_CDN = 'auto';
 
-    load();
-    expect(document.porscheDesignSystem.cdn).toBe('https://cdn.ui.porsche.com');
-  });
+      load();
+      expect(document.porscheDesignSystem.cdn.url).toBe('https://cdn.ui.porsche.com');
+    });
 
-  it('should for non .cn top level domain set document.porscheDesignSystem.cdn="https://cdn.ui.porsche.com"', () => {
-    Object.defineProperty(window, 'location', {
-      writable: true,
-      value: { origin: 'https://shop.porsche.com' },
+    it('should for non .cn top level domain set document.porscheDesignSystem.cdn.url="https://cdn.ui.porsche.com"', () => {
+      Object.defineProperty(window, 'location', {
+        writable: true,
+        value: { origin: 'https://shop.porsche.com' },
+      });
+
+      load();
+      expect(document.porscheDesignSystem.cdn.url).toBe('https://cdn.ui.porsche.com');
+
+      Object.defineProperty(window, 'location', {
+        writable: true,
+        value: { origin: 'https://shop.porsche.de' },
+      });
+
+      load();
+      expect(document.porscheDesignSystem.cdn.url).toBe('https://cdn.ui.porsche.com');
     });
 
-    load();
-    expect(document.porscheDesignSystem.cdn).toBe('https://cdn.ui.porsche.com');
+    it('should for window.PORSCHE_DESIGN_SYSTEM_CDN="cn" set document.porscheDesignSystem.cdn.url="https://cdn.ui.porsche.cn"', () => {
+      window.PORSCHE_DESIGN_SYSTEM_CDN = 'cn';
 
-    Object.defineProperty(window, 'location', {
-      writable: true,
-      value: { origin: 'https://shop.porsche.de' },
+      load();
+      expect(document.porscheDesignSystem.cdn.url).toBe('https://cdn.ui.porsche.cn');
     });
 
-    load();
-    expect(document.porscheDesignSystem.cdn).toBe('https://cdn.ui.porsche.com');
+    it('should for .cn top level domain set document.porscheDesignSystem.cdn.url="https://cdn.ui.porsche.cn"', () => {
+      Object.defineProperty(window, 'location', {
+        writable: true,
+        value: { origin: 'https://shop.porsche.cn' },
+      });
+
+      load();
+      expect(document.porscheDesignSystem.cdn.url).toBe('https://cdn.ui.porsche.cn');
+    });
   });
 
-  it('should for window.PORSCHE_DESIGN_SYSTEM_CDN="cn" set document.porscheDesignSystem.cdn="https://cdn.ui.porsche.cn"', () => {
-    window.PORSCHE_DESIGN_SYSTEM_CDN = 'cn';
+  // TODO: enable once backwards compatibility with older versions isn't needed anymore
+  xdescribe('for flat structure', () => {
+    it('should for window.PORSCHE_DESIGN_SYSTEM_CDN="auto" set document.porscheDesignSystem.cdn="https://cdn.ui.porsche.com"', () => {
+      window.PORSCHE_DESIGN_SYSTEM_CDN = 'auto';
 
-    load();
-    expect(document.porscheDesignSystem.cdn).toBe('https://cdn.ui.porsche.cn');
-  });
+      load();
+      expect(document.porscheDesignSystem.cdn).toBe('https://cdn.ui.porsche.com');
```

Comment:
don't you need to check against `.cdn.url`?

Reply:
no, that's the skipped describe block for the old and hopefully future implementation